main.go

```diff
@@ -113,6 +125,26 @@ func main() {
 		)
 	}
 
+	var tcpAppWorkflow cfWorkflow.CfWorkflow
+	var tcpAppCmdGenerator cfCmdGenerator.CfCmdGenerator
```

Comment:
Personal preference from my end would be to just use `tcp` as a prefix rather than `tcpApp` in most places to shorten variable names.

Reply:
👍